examples/angular/angular.json

```diff
@@ -41,7 +41,7 @@
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
             "assets": ["src/favicon.ico", "src/assets"],
-            "styles": ["src/styles.scss"],
+            "styles": ["src/styles.scss", "@aws-amplify/ui-angular/theme.css"],
```

Comment:
I know I wrote this 🤣 , but I think we should put `@aws-amplify/ui-angular/theme.css` first so that app styles win over our styles.